packages/language-server/test/check/diagnostics.test.ts

```diff
@@ -18,7 +18,7 @@ describe('Astro Check', async () => {
 		const [{ diagnostics }] = await checker.getDiagnostics();
 
 		expect(diagnostics.length, 'Expected diagnostics to be empty').to.equal(0);
-	});
+	}).timeout(10000);
```

Comment:
I'm not sure why but the first test in this file is taking a very long time. The tests in this file are kinda faulty in general and will be removed in the future (we don't need to test through AstroCheck anymore)